scripts/stage-release.sh

```diff
@@ -101,12 +101,15 @@ cleanup() {
 }
 
 setVersion() {
-  local VERSION=$1
-  mvn -q -B versions:set -DnewVersion="${VERSION}" -DgenerateBackupPoms=false -DprocessAllModules=true
+  local NEW_VERSION=$1
+  local CURRENT_VERSION
+
+  CURRENT_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=project.version -q -DforceStdout)
```

Comment:
The other assignment to `CURRENT_VERSION` at line 122 can be removed by the look of it, this function is the only user of `CURRENT_VERSION`